src/OSPSuite.Core/Domain/PKAnalyses/PKParameter.cs

```diff
@@ -31,5 +32,11 @@ public virtual string DisplayName
       public virtual IDimension Dimension { get; set; }
 
       public virtual PKParameterMode Mode { get; set; }
+
+      public virtual Unit DisplayUnit
```

Comment:
THis will only be set from R. Same behavior as before otherwise by using the default unit of a dimension